Library/Martial Arts/Martial Arts Skills.skl

```diff
@@ -1049,6 +1049,18 @@
 				"Technique"
 			]
 		},
+		{
+			"type": "skill",
+			"id": "f5c71028-8918-4424-bd42-f1eb53a16604",
+			"name": "Expert Skill (Hopology)",
+			"reference": "B193,MA:56",
```

Comment:
The page ref of MA:56 doesn't look right. Should this be MA56 instead?

Reply:
I fixed that.